classes/scripts/runner.php

```diff
@@ -608,7 +606,7 @@ public static function enableAutorun($name)
 						{
 							$autorunner->writeError($exception->getMessage());
 
-							exit($exception->getCode());
+							exit(2);
```

Comment:
It's a change in behavior that may affect the result of a test suite.
It's clear that the actual result is wrong, and this update will correct the result, but we need to note this for the next version